### ISSUE TITLE
[Modded] Massive lag spikes close to builds/structures.

### ISSUE THREAD

**Issue by AreYSerious** [anegostudios/VintageStory-Issues#3566]
### Game Version

v1.19.3

### Platform

Windows

### Modded

Modded

### SP/MP

Singleplayer

### Description

What could cause those massive lag spikes.
They happen very often and either have something to do with blocks or worldgen.
No idea tho.

  169,88ms, 1823 calls, avg 93,19 us/call: initbe-Generic
  
What does initbe-Generic mean how do I know what causes those insane lags.

### How to reproduce

Download this map
[350MB Map](https://www.dropbox.com/scl/fi/r3yzopc18m960tmzfra2z/Lag-spikes-at-1222-80-4426-flying-around.zip?rlkey=tejextb957hlvvvudq53tm96r&dl=0)
go to the coords 
`/tp 1222 80 4426`
fly around or walk around there are 4 large builds there which cause massive freeze frames. why is that?

### Screenshots

![2024-02-07_17-35-37](https://github.com/anegostudios/VintageStory-Issues/assets/100879715/cffd810a-90e7-4a7b-be7f-3b2c09ea91d6)

### Logs

```
7.2.2024 17:23:35 [Notification] A tick took 207 ms
207,00ms,  call : all
  169,88ms, 1823 calls, avg 93,19 us/call: initbe-Generic
  8,58ms, 10 calls, avg 857,99 us/call: initbe-PlantContainer
  6,27ms, 1568 calls, avg 4,00 us/call: initbe-Chisel
  5,02ms, 128 calls, avg 39,19 us/call: controlledphysics
    4,56ms, 239 calls, avg 19,08 us/call: collision
  2,00ms, 1 call : initbe-RiftWard
  1,09ms, 2 calls, avg 545,60 us/call: rend3D-rets-op
  0,94ms, 1 call : gmleVintagestory.Client.NoObf.HudElementBlockAndEntityInfo
  0,87ms, 1 call : Before-ree
  0,69ms, 1 call : initbe-Sign
  
7.2.2024 17:42:17 [Notification] A tick took 2704,47 ms
2704,47ms,  call : all
  2659,73ms, 128 calls, avg 20779,12 us/call: entityAgent-ticking
  31,97ms, 6 calls, avg 5327,62 us/call: readpacket33
  0,99ms, 2 calls, avg 494,60 us/call: particles-tick
  0,93ms, 1 call : readpacket60
  0,80ms, 2 calls, avg 397,85 us/call: rend3D-rets-op
  0,67ms, 1 call : Before-ree
  0,56ms, 667 calls, avg 0,84 us/call: Opaque-beanimutil
  0,50ms, 127 calls, avg 3,96 us/call: controlledphysics
  0,45ms, 1 call : rend3D-ret-opnc
```

**Comment by AreYSerious** [comment]
The mod doesn't add any new blocks, FYI

**Comment by MochaModifier** [comment]
Found this issue after updating and experiencing severe lag spikes in single player.

If we're having the same issues, I'm not so sure this is tied to a mod.
I tried making a new world with no mods enabled; and would still experience the game lagging/locking up.

Reducing view distance seems to help, so may be involved. 
The setting I had in 1.8.x worked without issue; but in 1.9.x I now get these spikes.

Try adjusting your view distance lower and see if that helps.

Still testing things on my end.
